index.html

```diff
@@ -15,13 +15,77 @@
       href="https://fonts.googleapis.com/css2?family=Montserrat:wght@300&display=swap"
       rel="stylesheet"
     />
+    <link rel="stylesheet" href="https://use.fontawesome.com/releases/v5.6.0/css/all.css"
+      integrity="sha384-aOkxzJ5uQz7WBObEZcHvV5JvRW3TUc2rNPA7pe3AwnsUohiw1Vj2Rgx2KSOkF5+h" crossorigin="anonymous">
 
     <link rel="stylesheet" href="styles.css" />
   </head>
   <body>
+    <header>
+      <nav class="navbar navbar-expand-lg navbar-dark bg-dark">
+        <div class="container">
+          <a class="navbar-brand" href="/bookGoals-frontend/index.html">BookGoals</a>
```

Comment:
This should be a relative path so the site will work regardless of folder structure.

```html
           <a class="navbar-brand" href="./index.html">BookGoals</a>
```